src/test/java/core/basesyntax/service/FileReaderServiceImplTest.java

```diff
@@ -0,0 +1,50 @@
+package core.basesyntax.service;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.fail;
+
+import java.util.List;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class FileReaderServiceImplTest {
+    private static FileReaderService fileReaderService;
+
+    @BeforeClass
+    public static void beforeClass() {
+        fileReaderService = new FileReaderServiceImpl();
+    }
+
+    @Test
+    public void read_validFilePath_ok() {
+        String validFilePath =
+                this.getClass().getClassLoader().getResource("FileReaderTestFile.csv").getPath();
+        List<String> expectedResultList = List.of(
+                "b,banana,0",
+                "b,apple,10",
+                "s,banana,15",
+                "p,banana,5",
+                "r,apple,20");
+        List<String> actualResultList = fileReaderService.read(validFilePath);
+        assertEquals(expectedResultList, actualResultList);
+    }
+
+    @Test(expected = RuntimeException.class)
+    public void read_invalidFilePath_notOk() {
+        String invalidFilePath =
+                "/src/test/resources/testFiles.csv";
+        fileReaderService.read(invalidFilePath);
+    }
+
+    @Test
```

Comment:
```suggestion
    @Test(expected = RuntimeException.class)
```
lets make your code consistent